src/WebJobs.Script.WebHost/Diagnostics/DiagnosticEventTableStorageRepository.cs

```diff
@@ -136,11 +142,13 @@ internal virtual async Task FlushLogs(CloudTable table = null)
             }
             catch (Exception ex)
             {
-                _logger.LogError(ex, $"Unable to create table '{table.Name}' after {_tableCreationRetries} retries. Aborting write operation {ex}");
-
+                _logger.LogError(ex, $"Unable to get table reference or create table. Aborting write operation {ex}");
```

Comment:
I don't think this error format is correct - don't need the "ex" param. LogError already logs the exception details.